games/TicTacToe/contract/src/main.sw

```diff
@@ -0,0 +1,98 @@
+contract;
+
+use core::*;
+use std::{address::Address, chain::auth::msg_sender, hash::sha256, storage::{StorageMap, get, store}};
+
+use data_structures::{Game, Players, Winner, core::ops::Eq};
+
+storage {
+    game: Game,
+    player_turn: u64,
+    map: StorageMap<u64,
```

Comment:
It's worth considering how you're going to write an algorithm to efficiently calculate if there is a win on the board, when deciding how to represent the map. Something organized 3x3 will allow you to reason about indices directly.